Cabal/Distribution/Backpack/UnifyM.hs

```diff
@@ -390,16 +481,29 @@ convertInclude (ComponentInclude {
     -- are mapped to the same name, the intent is to merge them
     -- together.  But they are *functions*, so @B as X, B as Y@ is
     -- illegal.
-    let insertDistinct m (k,v) =
-            if Map.member k m
-                then error ("Duplicate requirement renaming " ++ display k)
-                else return (Map.insert k v m)
-    req_rename <- foldM insertDistinct Map.empty =<<
-                      case req_rns of
-                        DefaultRenaming -> return []
-                        -- Not valid here, but whatever
-                        HidingRenaming _ -> error "Cannot use hiding in requirement renaming"
-                        ModuleRenaming rns -> return rns
+
+    req_rename_list <-
+      case req_rns of
+        DefaultRenaming -> return []
+        -- Not valid here, but whatever
```

Comment:
Move comment to next line just to make it absolutely clear which case it applies to?